src/Cmts.ml

```diff
@@ -56,7 +56,7 @@ end = struct
       (List.find_map roots ~f:(fun root ->
            if Itv.contains root elt then
              Hashtbl.find_and_call tbl root
-               ~if_found:(fun children ->
+               ~if_found:( fun children ->
```

Comment:
Note that originally the space for indicate-multiline-delimiters is the one after `(fun`, that is, the whole of `(fun` is taken to be the delimiter since `fun` is not user code and must be followed by a space.

Reply:
I will revert it.